### PR TITLE
ci: add Markdown linter for docs

### PR DESCRIPTION
**Pull request by subhamkrai**
Adding Markdown linter for docs.

Closes: https://github.com/ceph/persistent-volume-migrator/issues/32
Signed-off-by: subhamkrai <srai@redhat.com>

### CODE REVIEW

File: .github/workflows/markdown-lint.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Markdown-lint
+on:
+  pull_request:
+
+jobs:
+  lint-changelog:
```
Comment (Madhu-1) [inline .github/workflows/markdown-lint.yaml:6]: we are not linking changelog here right? need to adjust this markdown names

File: .github/workflows/markdown-lint.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Lint changelog file
+      uses: docker://avtodev/markdown-lint:v1
+      with:
+        rules: '/lint/rules/changelog.js'
+        config: '/lint/config/changelog.yml'
```
Comment (Madhu-1) [inline .github/workflows/markdown-lint.yaml:17]: where are these files created?

File: .github/workflows/markdown-lint.yaml (hunk 1 above)
Comment (subhamkrai) [reply .github/workflows/markdown-lint.yaml]: copy-paste error.

File: .github/workflows/markdown-lint.yaml (hunk 2 above)
Comment (subhamkrai) [reply .github/workflows/markdown-lint.yaml]: these files are not required in our case. These files are config files for the linter.  Initially, I thought these are required fields. Thanks

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**